src/main/java/com/orasi/bluesource/Accounts.java

```diff
@@ -46,7 +46,25 @@
 	@FindBy(css = "div.btn.btn-secondary.btn-xs.quick-nav") private Button btnQuickNav;
 	@FindBy(xpath = "//a[contains(@ng-bind, 'n + 1')]") private List<Button> btnPages;
 	@FindBy(xpath = "//*[@id=\"project-list\"]/div/div[1]/div") private Button btnCloseQuickNav;
-
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[5]/div/button[1]") private Button btnNewProject;
+	@FindBy(id = "project_name") private Textbox txtProjectName;
+	@FindBy(id = "project_start_date") private Textbox txtProjectStart;
+	@FindBy(id = "project_end_date") private Textbox txtProjectEnd;
+	@FindBy(xpath = "//*[@id=\"new_project\"]/div[11]/input") private Button btnCreateProject;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[13]/div/div[1]/button[1]") private Button btnNewRole;
+	@FindBy(xpath = "//*[@id=\"new_role\"]/div[1]/div/div") private Button btnBill;
+	@FindBy(id = "role_role_type_id") private Listbox lstRoleType;
+	@FindBy(id = "role_name") private Textbox txtRoleName;
+	@FindBy(id = "role_max_resources") private Textbox txtMaxResources;
+	@FindBy(xpath = "//*[@id=\"new_role\"]/div[11]/input") private Button btnCreateRole;
+	@FindBy(xpath = "//*[@id=\"panel_body_3\"]/div/div/table") private Webtable tblProjectRoles;
+	@FindBy(id = "filled_role_employee_id") private Listbox lstAssignEmployee;
+	@FindBy(xpath = "//*[@id=\"new_filled_role\"]/div[6]/input") private Button btnCreateFilledRole;
+	@FindBy(xpath = "//*[@id=\"project-list\"]/div/div[2]/ul") private Element elmQuickNavProjectList;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[9]/div/h4/div/button") private Button btnProjectOptions;
+	@FindBy(xpath = "//*[@id=\"accordion\"]/div/div[9]/div/h4/div/ul") private Element elmProjectOptionList;
+	@FindBy(linkText = "Close Project") private Link lnkCloseProject;
```

Comment:
Some of your xpaths look a bit flaky here, lots of relying on the index of an element.  problem with that is that html usually changes a lot, and they may add more divs, etc, and then your xpaths break.  I generally try to find a unique identifier for the object, or if not, then parent or sibling element, and work my way back.  Can all do innertext contains, attribute value text contains, etc.  Lost of options.